src/cli/commands/files/cat.js

```diff
@@ -14,19 +15,16 @@ module.exports = {
 
   handler (argv) {
     const path = argv['ipfs-path']
-    utils.getIPFS((err, ipfs) => {
+
+    waterfall([
+      (cb) => utils.getIPFS(cb),
+      (ipfs, cb) => ipfs.files.cat(path, cb)
+    ], (err, file) => {
       if (err) {
         throw err
       }
 
-      ipfs.files.cat(path, onFile)
+      file.pipe(process.stdout)
```

Comment:
I don't remember making that decision, but happy to move all these `run-*` to modular async.